src/tests/ftest/util/command_utils_base.py

```diff
@@ -480,6 +482,26 @@ def create_yaml(self, filename=None):
             self.reset_yaml_data_updated()
         return create_yaml
 
+    # Makito added
+    def write_yaml(self, yaml_data):
+        """Write yaml to file from provided yaml data.
+
+        Args:
+            yaml_data (str): dictionary containing yaml file values.
+
+        Raises:
+            CommandFailure: if there is an error creating the yaml file
+
+        """
+        self.log.info("Writing yaml configuration file %s", self.filename)
+        try:
+            with open(self.filename, 'w') as write_file:
+                yaml.dump(yaml_data, write_file, default_flow_style=False)
+        except Exception as error:
+            raise CommandFailure(
```

Comment:
(pylint-raise-missing-from) Consider explicitly re-raising using the 'from' keyword